packages/core/src/lib/abstracts/abstract-validation.directive.ts

```diff
@@ -62,7 +62,9 @@ export class AbstractValidationDirective implements OnDestroy {
   get blueprints(): Validation.Blueprints {
     return {
       ...BLUEPRINTS,
-      ...(this._blueprints || this.parent.blueprints || this.config.blueprints || {}),
```

Comment:
Actually, it is not a bug. it is a design decision but we can change that. Let me think a little bit,

Reply:
Sure. I think at least module level blueprints should not be replaced since there could be many custom messages and if we need to simple change a message to a control for instance pattern validation could have its own message as per the control so I should be able to only update pattern validation message on that control level without affecting everything.